frontend/src/router/pages/ExportPage.jsx

```diff
@@ -0,0 +1,161 @@
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
+function ExportPage() {
+  return (
+    <div className="page export">
+      <p>Select the format of your export:</p>
+
+      <div className="row">
+        <div className="col-md-6 col-lg-5">
+          <fieldset className="mb-3">
+            <legend className="append-required">Export type</legend>
+
+            <div className="form-check">
+              <input
+                className="form-check-input"
+                type="radio"
+                name="type"
+                id="type-all-dates"
+                value="all-dates"
+              />
+              <label className="form-check-label" htmlFor="type-all-dates">
+                All dates
+              </label>
+            </div>
```

Comment:
All labels looked smaller than the mock (12px). Should it be 16px?

Reply:
It does look small, but that's how looks is in the bootstrap theme 🤷 
https://digitalspace.github.io/bcparks-bootstrap-theme/#forms

My understanding is that the Figma link is just wireframes and not an exact design mockup so I've just been using the default Bootstrap styles as much as possible.. But I don't think anything should be less than 16px in the app. I'll add an override to make the radio/checkbox label font bigger and check with design to see if there's a specific idea.

Reply:
Oh, I see the "label" size is supposed to be 0.75rem in the design tokens: https://www2.gov.bc.ca/gov/content/digital/design-system/foundations/design-tokens/glossary
But that is probably meant for text box labels and not checkbox and radio buttons. I'll make an override to leave those ones at 1rem 👍